src/OSPSuite.UI/Extensions/ChartControlExtensions.cs

```diff
@@ -30,5 +36,97 @@ public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartF
             xyDiagram.GetAllAxesY().Each(axis => axis.Title.Font = fontAxis);
          }
       }
+
+      private const string WATERMARK_ANNOTATION = "WATERMARK_ANNOTATION";
```

Comment:
Moved those guys into the ChartControlExtension files. We had two files doing the same thing